File: Mock/Data/Exception/CanNotFindPropertyException.cs
namespace Mock.Data.Exception
{
    /// <summary>
    /// 找不到数据异常
    /// </summary>
    public class CanNotFindPropertyException : TaistException
    {
        /// <summary>
        /// 构造找不到数据异常的新实例
        /// </summary>
        public CanNotFindPropertyException(string className, string propertyName) : base(string.Format("Can not find property {0} in class {1}", propertyName, className)) { }
    }
}

File: java/Taist.Framwork/src/Mock/Data/Internal/CannotFindPropertyException.java
package Mock.Data.Internal;

import Mock.Exception.TaistException;

public class CannotFindPropertyException extends TaistException {
	/**
	 * 
	 */
	private static final long serialVersionUID = -4443315128603165908L;

	public CannotFindPropertyException(String propertyName) {
		super("Can not find property named" + propertyName);
	}
}
